ta4j-core/src/main/java/org/ta4j/core/indicators/MACDIndicator.java

```diff
@@ -1,40 +1,60 @@
-/*
-  The MIT License (MIT)
-
-  Copyright (c) 2014-2017 Marc de Verdelhan & respective authors (see AUTHORS)
-
-  Permission is hereby granted, free of charge, to any person obtaining a copy of
-  this software and associated documentation files (the "Software"), to deal in
-  the Software without restriction, including without limitation the rights to
-  use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of
-  the Software, and to permit persons to whom the Software is furnished to do so,
-  subject to the following conditions:
-
-  The above copyright notice and this permission notice shall be included in all
-  copies or substantial portions of the Software.
-
-  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
-  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
-  FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
-  COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
-  IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
-  CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+/**
```

Comment:
I have changed the license header because it should not be interpreted as javadoc
https://stackoverflow.com/questions/43874446/why-does-intellij-idea-give-a-warning-that-this-file-javadoc-is-dangling

Reply:
Alternatively we could just remove one star like https://github.com/jfree/jfreechart/blob/master/src/main/java/org/jfree/chart/annotations/AbstractAnnotation.java. But i would prefer to stay this way because at the moment all files have this kind of license header

Reply:
I removed one star so I use this for license header:

```
/*
..
*/
```